### ISSUE TITLE
Integrating Game Database with Bump Explore

### ISSUE THREAD

**Issue by tjg17** [suyashkumar/bump-explore#3]
We'll need to talk more about how to get our SQLite database to play nicely with bump explore once we have launched the website. Any initial thoughts/ideas/concerns from your end?

**Comment by suyashkumar** [comment]
Yeah! I think we definitely need to think about making a RESTful API for me to grab data from the server. For example I should be able to go to /api/players/Tyler to get your game history! Honestly I wonder if I can just mirror the sqllite database and/or generally tap directly into it.
